### PR TITLE
Us012 add new role

### PR DESCRIPTION
**Pull request by matthewmcpolinkainos**

### CODE REVIEW

File: SQLScripts/FToPayRespects_JobRoles_InsertStatements.sql [hunk 1]
```diff
@@ -549,3 +549,21 @@ Setting Direction, Development and Accountability - Prioritise and communicate e
 Supporting and Delivering the Strategy - Articulates individual contribution to the wider Kainos objectives, clearly defines how individual personal performance can have an impact on the business, respects differences and promotes inclusion.
 Commerciality and Risk - Comprehends the need for the business to generate additional income and respects that costs need to be managed, considers and suggests ideas for improvements.
 Communicating and Influence - Involved at meetings, asking questions, remaining communicative, demonstrates influencing skills, demonstrates how to participate in stakeholder management.' WHERE JobRoleID = 9;
+<<<<<<< Updated upstream
```
Comment (felixmoore) [inline SQLScripts/FToPayRespects_JobRoles_InsertStatements.sql:552]: Looks like there are leftover merge conflict artefacts here

**Review by Emanity — APPROVED** [review]

**Review by felixmoore — APPROVED** [review]

**Review by lobola — CHANGES_REQUESTED** [review]
Looks great just one empty line you can remove :)

File: SQLScripts/FToPayRespects_TableCreation.sql [hunk 2]
```diff
@@ -6,18 +6,17 @@ DROP TABLE IF EXISTS JobCapability;
...
 
 CREATE TABLE JobBand (
-    JobBandID INT NOT NULL PRIMARY KEY,
+    JobBandID INT NOT NULL PRIMARY KEY auto_increment,
     JobBand VARCHAR(100) NOT NULL
 );
 
 CREATE TABLE JobCapability (
-	JobCapabilityID INT NOT NULL PRIMARY KEY,
+	JobCapabilityID INT NOT NULL PRIMARY KEY auto_increment,
```
Comment (jkelso662) [inline SQLScripts/FToPayRespects_TableCreation.sql:14]: Indentation is inconsistent

File: src/main/java/com/kainos/ea/db/JobRolesDAO.java [hunk 3]
```diff
@@ -1,32 +1,61 @@
...
+  @SqlQuery("SELECT DISTINCT JobDisciplineID, JobDiscipline FROM JobDiscipline")
+  @RegisterBeanMapper(JobDiscipline.class)
+  List<JobDiscipline> getJobDiscipline();
+
+  @SqlUpdate("INSERT INTO JobBand (JobBand, JobBandTraining) VALUES (:jobBand, :jobBandTraining );")
+  boolean addJobBand(@Bind("jobBand") String jobBand, @Bind("jobBandTraining") String jobBandTraining);
+
+  @SqlUpdate("INSERT INTO JobCapability (JobCapability) VALUES (:jobCapability );")
+  boolean addJobCapability(@Bind("jobCapability") String jobCapability);
+
```
Comment (jkelso662) [inline src/main/java/com/kainos/ea/db/JobRolesDAO.java:60]: Not a huge issue but I'd normally not have any spaces at the end of classes and closing braces.  And as long as you are being consistent.

File: src/main/java/com/kainos/ea/objects/JobBand.java [hunk 4]
```diff
@@ -6,30 +6,39 @@
 public class JobBand {
 
   private int jobBandId;
-  private JobBand jobBand;
+  private String jobBand;
+  private String jobBandTraining;
 
   public JobBand() {
```
Comment (jkelso662) [inline src/main/java/com/kainos/ea/objects/JobBand.java:12]: Is there a use case for having an empty constructor?

**Review by lobola — APPROVED** [review]

File: src/main/java/com/kainos/ea/objects/JobBand.java (hunk 4 above)
Comment (jkelso662) [reply src/main/java/com/kainos/ea/objects/JobBand.java:12]: Just realised these are probably your DB objects which maybe do need to have empty constructors?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Very good! Only minor things. A couple of styling issues. Also just checking R.E. empty constructors - otherwise great. Very close to simply approving this one!

File: src/main/java/com/kainos/ea/objects/JobBand.java (hunk 4 above)
Comment (matthewmcpolinkainos) [reply src/main/java/com/kainos/ea/objects/JobBand.java:12]: I believe we are using the empty constructors in some of our tests

**Pull request merged**